src/client/style/ResultsSearchbar.css

```diff
@@ -2,9 +2,10 @@
   width: 85%;
   height: 100%;
   margin: auto;
-  padding: 14px 54px;
+  padding: 1em 4em;
   font-family: 'Open Sans', Arial, sans-serif;
   font-size: 18px;
+  box-sizing: border-box;
   border: none;
   border-radius: 90px;
```

Comment:
nit: border radius 90px from my POV is large enough value to be problematic on small screens. Consider adaptive units. box-shadow 3 and 8px might work ok